internal/task/scripts/shell/format.sh

```diff
@@ -9,11 +9,14 @@ percent=$2
 chunkfile_size=$3
 chunkfile_pool_dir=$4
 chunkfile_pool_meta_path=$5
+chunkfile_block_size=$6
 
 mkdir -p $chunkfile_pool_dir
 $binary \
   -allocatePercent=$percent \
   -fileSize=$chunkfile_size \
   -filePoolDir=$chunkfile_pool_dir \
   -filePoolMetaPath=$chunkfile_pool_meta_path \
-  -fileSystemPath=$chunkfile_pool_dir
+  -fileSystemPath=$chunkfile_pool_dir \
```

Comment:
Will the command report invalid option if user use the old container image which not support specify the `fileSystemPath`?

Reply:
below `-undefok` is a workaround for this, older format tools won't report error.